native/swift/Sources/wordpress-api/WordPressLoginClient.swift

```diff
@@ -0,0 +1,119 @@
+import Foundation
+import WordPressAPIInternal
+
+#if os(Linux)
+import FoundationNetworking
+#endif
+
+public final class WordPressLoginClient {
```

Comment:
BTW, there is not much going on in this type. Can we use `UniffiWpLoginClient` directly? You'll need to add the `loginURL(...)` and `credentials(...)` helper functions to it, but I don't think that's a terrible thing to do.

Reply:
I'd be fine if we get there eventually, but the name is wrong – nobody consuming this library should ever see `UniffiWpLoginClient`. At the very least we'll have to typealias it